src/index.js

```diff
@@ -12,6 +12,12 @@ import LineStream from './line-stream';
 import ParseStream from './parse-stream';
 import Parser from './parser';
 
+export {
```

Comment:
I can just change `export default` to `module.exports` to do the same thing. I think thats cleaner, any objection to that?